src/domains/agents/communication/proactive_agent.py

```
@@ -345,45 +615,337 @@ async def _generate_proactive_message(self, message_type: str, belief_content: D
+                    meeting_time = last_meeting.get("time", "")
+                    attendees = last_meeting.get("attendees", [])
+                    
+                    # Format meeting time for display
+                    try:
+                        from datetime import datetime
+                        if meeting_time:
+                            dt = datetime.fromisoformat(meeting_time.replace('Z', '+00:00'))
+                            formatted_time = dt.strftime("%A at %I:%M %p")
+                        else:
+                            formatted_time = "the scheduled time"
+                    except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)